.ci/setup_env_fedora.sh

```diff
@@ -10,17 +10,9 @@ set -e
 cidir=$(dirname "$0")
 source /etc/os-release || source /usr/lib/os-release
 source "${cidir}/lib.sh"
-TEST_CGROUPSV2="${TEST_CGROUPSV2:-false}"
-
 echo "Install chronic"
 sudo -E dnf -y install moreutils
 
-if [ "${TEST_CGROUPSV2}" == "true" ]; then
-	echo "Install podman"
-	version=$(get_test_version "externals.podman.version")
-	sudo -E dnf -y install podman-"${version}"
```

Comment:
I think we should still install podman, but do it in the same way we do for docker and do not rely on a specific version anymore.

We still can use podman together with osbuilder and this should be, somehow, supported and tested.

Reply:
@fidencio why we want to install podman? what is the benefit? what are we going to test?

Reply:
@fidencio we are installing docker for other purposes but I do not see the benefit of using podman when we have nothing to test with kata 2.x

Reply:
currently in kata 2.x, all these scripts have not been executed at any time, mainly because the CI flag that was used to run all these steps was used in kata 1.x but never used in kata 2.x

Reply:
Hmmm.  Don't we have the option of using osbuilder with either podman or docker?  I thought we had some test covering both.

If no, then we can remove it now and, maybe, later on, add a test to ensure we actually can use osbuilder with docker.